regulations/tasks.py

```diff
@@ -72,10 +74,16 @@ def submit_comment(self, body, files):
     except (ClientError, RequestException) as exc:
         logger.exception(exc)
         raise self.retry(exc=exc)
+    except Task.MaxRetriesExceededError as exc:
```

Comment:
Yes, it is. I have tested it. `MaxRetriesExceededError` will be thrown from inside the previous `except`, and will be caught here.